### ISSUE TITLE
git dependancy bug

### ISSUE THREAD

**Issue by Krukov** [yhino/pipenv-poetry-migrate#7]
Thanks for the tool! 

This tool has a wrong transformation for dependency like that 
```Pipfile
pipenv-poetry-migrate = {editable = true,git = "https://github.com/yhino/pipenv-poetry-migrate.git",ref = "master"}

```

**Comment by yhino** [comment]
Thanks for reporting the issue.
I tried to fix it with 0.1.3. Please check if it is fixed.

**Comment by Krukov** [comment]
@yhino  Thanks you, it works
